library/src/interactor_impl.cxx

```diff
@@ -319,6 +320,10 @@ class interactor_impl::internals
         self->SetViewOrbit(ViewType::VT_RIGHT, self);
         render = true;
         break;
+      case '5':
```

Comment:
you should add an interactive test for this

Reply:
Definitely, the codecov is also failing because there is not test

Reply:
I added the test cases for orthographic projection and updated the baseline